spec/system/lesson_project_submissions/like_spec.rb

```diff
@@ -51,4 +63,46 @@
       end
     end
   end
+
+  context 'when submission has no likes and user is duplicate' do
```

Comment:
Nit: "user is duplicate" might lead to some confusion with what we're testing here, is there something else that could communicate the intent better to anyone that needs to touch these tests in the future? 

```suggestion
  context 'When a user is gaming likes' do
```

Maybe too hostile? 😆